services/utils/__init__.py

```diff
@@ -0,0 +1,190 @@
+import json
+import sys
+import os
+import traceback
+import collections
+import pkg_resources
+from urllib.parse import urlencode
+from multidict import MultiDict
+from aiohttp import web
+from functools import wraps
+from typing import Dict
+import logging
+
+version = pkg_resources.require("metadata_service")[0].version
+
+METADATA_SERVICE_VERSION = version
+METADATA_SERVICE_HEADER = 'METADATA_SERVICE_VERSION'
+
+# The latest commit hash of the repository, if set as an environment variable.
+SERVICE_COMMIT_HASH = os.environ.get("BUILD_COMMIT_HASH", None)
+# Build time of service, if set as an environment variable.
+SERVICE_BUILD_TIMESTAMP = os.environ.get("BUILD_TIMESTAMP", None)
+
+# Setup log level based on environment variable
+log_level = os.environ.get('LOGLEVEL', 'INFO').upper()
+logging.basicConfig(level=log_level)
+
+
+async def read_body(request_content):
```

Comment:
a lot of the functions in this file are present in:
https://github.com/Netflix/metaflow-service/blob/master/services/metadata_service/api/utils.py
can you remove the duplicates?